app/(auth)/login/actions.ts

```diff
@@ -12,7 +12,6 @@ export async function login(formData: FormData) {
     email: formData.get('email') as string,
     password: formData.get('password') as string,
   };
-
   const { error } = await supabase.auth.signInWithPassword(data);
```

Comment:
it'd be easier if we get as args `(data: z.infer<typeof LoginSchema>)`, then remove the `data` object and just pass `data` from args straight to the supabase method